crypto/src/openssl.rs

```diff
@@ -210,6 +211,23 @@ impl Crypto for OpensslCrypto {
         Ok(super::EcdsaSig { r, s })
     }
 
+    #[cfg(feature = "alias_key_from_file")]
```

Comment:
I kind of like the idea of always having the alias key come from a file. Should we make this the only option?

Reply:
Removed.